src/localization/en.json

```diff
@@ -520,6 +526,7 @@
         "library_f2p": "Show played free to play games and demos in library",
         "inventory_market_text": "Show Market price on inventory page",
         "send_age_info": "Automatically send age verification and skip mature content warnings",
+        "showdeckcompat": "Always show Steam Deck compatibility status",
```

Comment:
As you said, the move of the node should also be included here, but it has to be concise such that the text is not too long

Reply:
It's kinda difficult to label this feature in a few words... might as well copy my comment over as a tooltip or something. I'll think about it some more.

Reply:
I settled on "Show Steam Deck compatibility ratings in the features list (and fetch the data when it's missing)". How does that sound?

Reply:
Sounds good